manager/integration/Dockerfile

```diff
@@ -1,22 +1,32 @@
 FROM registry.suse.com/bci/python:3.9
 
 ARG KUBECTL_VERSION=v1.17.0
```

Comment:
nit: Can we make the kubectl version same across the test infra? Like at other places, we are using 1.20.2 or we can use the latest stable version.

Reply:
kubectl binary doesn't provide a download url with `latest` tag. We have to manually do `$(curl -L -s https://dl.k8s.io/release/stable.txt)` to know the latest version, and then use `curl -LO https://dl.k8s.io/release/${LATEST_VERSION}/bin/linux/amd64/kubectl` to download the latest one. 

But I think we can update the version and make it consistent across the 2 Dockerfiles in longhorn-tests.

Reply:
Created https://github.com/longhorn/longhorn/issues/6485 for this.